src/Symfony/Bridge/Twig/Tests/Extension/TranslationExtensionTest.php

```diff
@@ -89,18 +89,35 @@ public function getTransTests()
             array('{% trans into "fr"%}Hello{% endtrans %}', 'Hello'),
 
             // transchoice
-            array('{% transchoice count from "messages" %}{0} There is no apples|{1} There is one apple|]1,Inf] There is %count% apples{% endtranschoice %}',
```

Comment:
as this is non-single line, it shall has comma after last item, yet it was not one item per line as well. let us normalize